dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/settings/SettingsWrapper.java

```diff
@@ -124,24 +126,46 @@ public String getConfiguredLocaleName(String localeCode) {
         return configuredLocales.get(localeCode);
     }
 
+    public Map<String, String> getConfiguredAboutUrls() {
+        if (configuredAboutUrls == null) {
+            initConfiguredAboutUrls();
+        }
+        return configuredAboutUrls;
+    }
+
     // -------------------- PRIVATE --------------------
 
     private void initLocaleSettings() {
+        configuredLocales = initMapFromJson(SettingsServiceBean.Key.Languages, "locale", "title");
+    }
 
-        configuredLocales = new LinkedHashMap<>();
+    private void initConfiguredAboutUrls() {
+        configuredAboutUrls = initMapFromJson(SettingsServiceBean.Key.NavbarAboutUrl, "url", "title");
+    }
 
+    private Map<String, String> initMapFromJson(SettingsServiceBean.Key settingKey, String mapKey, String mapValue) {
+        Map<String, String> map = new LinkedHashMap<>();
         try {
-            JSONArray entries = new JSONArray(getSettingValue(SettingsServiceBean.Key.Languages.toString()));
+            JSONArray entries = new JSONArray(getSettingValue(settingKey.toString()));
             for (Object obj : entries) {
                 JSONObject entry = (JSONObject) obj;
-                String locale = entry.getString("locale");
-                String title = entry.getString("title");
-
-                configuredLocales.put(locale, title);
+                String key = entry.getString(mapKey);
+                String value = getLocaleAwareString(entry, mapValue);
```

Comment:
I don't understand where this comes from. Why would we have to guess key-value format here?

Reply:
If this is just for `NavbarAboutUrl` processing then I would rather have a specific `init` method for this than `parse` method hidden inside this 'generic' method implementation.

Reply:
It's not guessing, it's constructing language specific keys - `title.pl`, `title.en`, etc... This is to allow defining values for multiple languages if necessary. Alternatively a default value can also be defined without the suffix - `title`. It's a generalization of practices already used in settings. No guessing here - we take the language from currently selected locale, and if there is a key for that language, we use a value from that key. If locale specific key is not defined, we use the default value.